packages/botonic-react/src/components/multichannel/multichannel-button.jsx

```diff
@@ -56,15 +56,30 @@ export const MultichannelButton = props => {
     return text
   }
 
+  const truncateText = (text, maxLength, ellipsis = '...') => {
+    if (text.length > maxLength) {
+      return text.substring(0, maxLength - ellipsis.length) + ellipsis
+    }
+    return text
+  }
+
   if (isWhatsapp(requestContext)) {
-    if (hasUrl()) {
-      return `${getText()}: ${getUrl()}`
-    } else if (hasPath() || hasPayload()) {
-      const text = getText()
-      increaseCurrentIndex()
-      return `${text}`
-    } else if (hasWebview()) return <Button {...props}>{getText()}</Button>
-    else return <Button {...props}>{props.children}</Button>
+    const asText = props.asText == null ? true : props.asText
```

Comment:
`asText` is a `boolean` prop and by default I want it to be `true` (to keep the actual behaviour).
If I do as you suggest: `const asText = props.asText ? true : props.asText`, if `asText` is set to `false` in the first place I will force it to `true` eitherway, and that's why I've done it this way (even with the `==` instead of the `===`), because this way I can check if is `null` or `undefined` and then assign it to `true` if that's the case.

Reply:
Right, I understand you want to keep the value if it's defined, otherwise you're setting it to `true`. If that's the case, you could use this simplification:
```
const asText = props.asText ?? true
```

Reply:
Totally agree with this new suggestion, thanks! Refactored!